### DISCUSSION TITLE
Data Importer Installation issue - Cannot open after installation

### DISCUSSION THREAD

**Discussion by GabrielBarcia** [firefly-iii/firefly-iii#5796] [Firefly III] [answered]
Hi,

I tried to install FIDI using this guide
https://docs.firefly-iii.org/data-importer/install/self_hosted/

The composer finish with no errors
```
Package manifest generated successfully.
56 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
> @php artisan key:generate --ansi
Application key set successfully.
```

I checked the .env variable, and only added the URL for the FF3 app

But when I try to access the application, I get a "Not Found" error

![image](https://user-images.githubusercontent.com/4810944/154807686-4b6d6d07-7ecb-4512-931c-f94b16c55a57.png)

I notice that is redirecting to /token, then get error. Tried to flush it, but also same error is displayed back.

```
gabriel@ubuntu:/var/www/FireFly3/data-importer$ curl http://ff3localdataimporter.fin
<!DOCTYPE html>
<html>
    <head>
        <meta charset="UTF-8" />
        <meta http-equiv="refresh" content="0;url='http://ff3localdataimporter.fin/token'" />

        <title>Redirecting to http://ff3localdataimporter.fin/token</title>
    </head>
    <body>
        Redirecting to <a href="http://ff3localdataimporter.fin/token">http://ff3localdataimporter.fin/token</a>.
    </body>
</html>

```

Not sure what else to do. Can I get some help? :)

Thanks
Gabriel

**Comment by JC5** [comment, marked as answer]
It looks like Apache doesn't accept the htaccess file. Try `AllowOverride All` in the vhost configuration.

**Reply by GabrielBarcia** [reply]
Yes, this was the problem. After adding the config to the vhost it started working. Thank you
